Finjector.Web/ClientApp/src/pages/Landing.tsx

```diff
@@ -15,22 +16,34 @@ const Landing = () => {
 
   const savedCharts = useGetSavedCharts();
 
+  const queryStatus: FinQueryStatus = {
+    isError: savedCharts.isError,
+    isInitialLoading: savedCharts.isInitialLoading,
+    error: savedCharts.error,
+  };
+
```

Comment:
Could probably use a helper, `const queryStatus = getQueryStatus(savedCharts)`.  Not that it's difficult to transfer these variables but would help and make it more future proof too.